### PR TITLE
`azurerm_kubernetes_cluster` - support for the `multiple_standard_load_balancers_enabled` property

### PR DESCRIPTION
**Pull request by ms-henglu**

### CODE REVIEW

File: internal/services/containers/kubernetes_cluster_resource.go [hunk 1]
```diff
@@ -3029,6 +3041,10 @@ func flattenKubernetesClusterNetworkProfile(profile *managedclusters.ContainerSe
 			lb["outbound_ports_allocated"] = v
 		}
 
+		if v := lbp.EnableMultipleStandardLoadBalancers; v != nil {
+			lb["multiple_standard_load_balancer_enabled"] = v
```
Comment (katbyte) [inline internal/services/containers/kubernetes_cluster_resource.go:3045]: this should be plural? as the property is + it doesn't make sense without a s?
```suggestion
		if v := lbp.EnableMultipleStandardLoadBalancers; v != nil {
			lb["multiple_standard_load_balancers_enabled"] = v
```

File: website/docs/r/kubernetes_cluster.html.markdown [hunk 2]
```diff
@@ -654,6 +654,8 @@ A `load_balancer_profile` block supports the following:
 
 ~> **Note:** `managed_outbound_ipv6_count` requires dual-stack networking. To enable dual-stack networking the Preview Feature `Microsoft.ContainerService/AKS-EnableDualStack` needs to be enabled and the Resource Provider re-registered, see [the documentation](https://docs.microsoft.com/azure/aks/configure-kubenet-dual-stack?tabs=azure-cli%2Ckubectl#register-the-aks-enabledualstack-preview-feature) for more information.
 
+* `multiple_standard_load_balancer_enabled` - (Optional) Is multiple standard load balancers per AKS cluster enabled?
```
Comment (katbyte) [inline website/docs/r/kubernetes_cluster.html.markdown:657]: grammer
```suggestion
* `multiple_standard_load_balancer_enabled` - (Optional) Are multiple standard load balancers for the cluster enabled?
```

**Review by katbyte — CHANGES_REQUESTED** [review]

**Review by stephybun — CHANGES_REQUESTED** [review]
Test failure:
```
------- Stdout: -------
=== RUN   TestAccKubernetesCluster_multipleLoadBalancer
=== PAUSE TestAccKubernetesCluster_multipleLoadBalancer
=== CONT  TestAccKubernetesCluster_multipleLoadBalancer
testcase.go:110: Step 3/4 error: After applying this test step, the plan was not empty.
stdout:
Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
~ update in-place
Terraform will perform the following actions:
# azurerm_kubernetes_cluster.test will be updated in-place
~ resource "azurerm_kubernetes_cluster" "test" {
id                                  = "/subscriptions/*******/resourceGroups/acctestRG-aks-230328075412883011/providers/Microsoft.ContainerService/managedClusters/acctestaks230328075412883011"
name                                = "acctestaks230328075412883011"
tags                                = {}
# (24 unchanged attributes hidden)
~ network_profile {
# (10 unchanged attributes hidden)
~ load_balancer_profile {
~ multiple_standard_load_balancers_enabled = true -> false
# (7 unchanged attributes hidden)
}
}
# (3 unchanged blocks hidden)
}
Plan: 0 to add, 1 to change, 0 to destroy.
--- FAIL: TestAccKubernetesCluster_multipleLoadBalancer (876.03s)
FAIL
```

**Comment by ms-henglu** [comment]
Hi all,

Thanks for the code review, but I got confirmation from the service team that this feature is not available yet, so I'll close this PR for now.

**Pull request closed without merging**